readme.md

```diff
@@ -182,20 +191,25 @@ includes information about [deployment use cases](https://opensource.zalando.com
-- Service Backends
-- Route Definitions
-- Data Sources: [eskip file](https://pkg.go.dev/github.com/zalando/skipper/eskipfile), [etcd](https://pkg.go.dev/github.com/zalando/skipper/etcd), [Kubernetes](https://pkg.go.dev/github.com/zalando/skipper/dataclients/kubernetes), [Route string](https://pkg.go.dev/github.com/zalando/skipper/dataclients/routestring)
-- [Circuit Breakers](https://pkg.go.dev/github.com/zalando/skipper/filters/circuit)
-- Extending It with Customized [Predicates](https://opensource.zalando.com/skipper/reference/predicates/), [Filters](https://opensource.zalando.com/skipper/reference/filters/), can be done by [Plugins](https://opensource.zalando.com/skipper/reference/plugins/) or [Lua Scripts](https://opensource.zalando.com/skipper/reference/scripts/)
 - [Predicates](https://opensource.zalando.com/skipper/reference/predicates/) - additional predicates to match a route
-- [Proxy Packages](https://pkg.go.dev/github.com/zalando/skipper/proxy)
+- Service [Backends](https://opensource.zalando.com/skipper/reference/backends/)
+- Route Definitions fetched by dataclients:
+   - [route string](https://opensource.zalando.com/skipper/data-clients/route-string/)
+   - [eskip file](https://opensource.zalando.com/skipper/data-clients/eskip-file/)
+   - [remote eskip](https://opensource.zalando.com/skipper/data-clients/eskip-remote/)
+   - [etcd](https://opensource.zalando.com/skipper/data-clients/etcd/)
+   - [kubernetes](https://opensource.zalando.com/skipper/data-clients/kubernetes/)
+- [Circuit Breakers](https://pkg.go.dev/github.com/zalando/skipper/filters/circuit)
+- Extending It with Custom [Predicates](https://opensource.zalando.com/skipper/tutorials/development/#predicates), [Filters](https://opensource.zalando.com/skipper/tutorials/development/#filters), can be done by [building your own proxy](https://opensource.zalando.com/skipper/tutorials/built-your-own/), [Plugins](https://opensource.zalando.com/skipper/reference/plugins/) or [Lua Scripts](https://opensource.zalando.com/skipper/reference/scripts/)
+- [Proxy Package](https://pkg.go.dev/github.com/zalando/skipper/proxy)
 - [Logging](https://pkg.go.dev/github.com/zalando/skipper/logging) and [Metrics](https://pkg.go.dev/github.com/zalando/skipper/metrics)
-- Performance Considerations
+- [Operations guide](https://opensource.zalando.com/skipper/operation/operation/)
+- [Authentication and Authorization](https://opensource.zalando.com/skipper/reference/filters/#authentication-and-authorization)
+- [Load Shedders](https://opensource.zalando.com/skipper/reference/filters/#load-shedding)
 - [Rate Limiters](https://pkg.go.dev/github.com/zalando/skipper/filters/ratelimit)
-- [Opentracing plugin](https://github.com/skipper-plugins/opentracing/) or extend [create your own](https://opensource.zalando.com/skipper/reference/plugins/#opentracing-plugins)
-- [WAF plugin](https://github.com/jptosso/coraza-waf) - Web Application Firewall filter for Skipper routes
```

Comment:
Why do we remove WAF plugin?

Reply:
Because it changed to not depend on skipper anymore